metrics/Contributing_Organizations.md

```diff
@@ -3,27 +3,24 @@
 Question: What is the number of contributing organizations?
 
 ## Description
-A description of what the metric is and what it captures.
-The first few sentences should match the description in the [metrics list](../activity-metrics-list.md).
-
+These are the contributions made by organizations that are seperate from the project being assessed. This does not include individual contributors to the project. For example, if a software company such as Google commited changes, it would count twards the number of contributing organizations, but an individual employee of Google would not count as being seperate. 
 ## Objectives
-Answer the question for why someone wants to measure this metric and what can be known with it.
+Volume of contributing organizations. This metric give insight into contributing orgainizations, and shows which organizations are contributing to the health of a project.

```

Comment:
Perhaps "Number of contributing organizations will provide a wide, granular view of all contributions by organization"